test/Microsoft.DotNet.ProjectModel.Tests/PackageDependencyProviderTests.cs

```diff
@@ -8,13 +10,39 @@
 using NuGet.Frameworks;
 using NuGet.Versioning;
 using Xunit;
-using System.IO;
 
 namespace Microsoft.DotNet.ProjectModel.Tests
 {
     public class PackageDependencyProviderTests : TestBase
     {
         [Fact]
+        public void GetDescriptionShouldUseTheProvidedPackagesDirectoryPath()
+        {
+            // Arrange
+            var provider = new PackageDependencyProvider(NuGetPathContext.Create("/foo/packages"), new FrameworkReferenceResolver("/foo/references"));
```

Comment:
NIT: break this line in less than 120 characters. Horizontal scrolling is pretty annoying.